test/spqr.jl

```diff
@@ -137,6 +139,15 @@ end
     @test all(iszero, (rank(spzeros(10, i)) for i in 1:10))
 end
 
+
+@testset "sparse" begin
+    A = I + sprandn(100, 100, 0.01)
+    Q = qr(A).Q
+    sQ = sparse(Q)
+    @test sQ == sparse(Matrix(Q))
+    @test sQ ≈ sparse(qr(Matrix(A)).Q)
```

Comment:
this is going to always fail, qr(::Sparse) is going to returned a permutated qr

Reply:
another problem is that the R and Q can be multiplied by -1 w/o changing the correctness.

Reply:
Yea, that was too quick. Thanks for fixing.